explore/src/main/scala/explore/config/ImagingModesTable.scala

```diff
@@ -0,0 +1,227 @@
+  private val FilterColumnId: ColumnId      = ColumnId("filter")
+  private val LambdaColumnId: ColumnId      = ColumnId("lambda")
+  private val DeltaLambdaColumnId: ColumnId = ColumnId("delta_lambda")
+  private val FovColumnId: ColumnId         = ColumnId("fov")
+
+  private val columnNames: Map[ColumnId, String] =
+    Map(
+      InstrumentColumnId  -> "Instrument",
+      TimeColumnId        -> "Time",
+      SNColumnId          -> "S/N",
+      FilterColumnId      -> "Filter",
+      LambdaColumnId      -> "λ",
+      DeltaLambdaColumnId -> "Δλ",
+      FovColumnId         -> "FoV"
+    )
+
+  private def column[V](
+    id:       ColumnId,
+    accessor: ImagingModeRowWithResult => V
+  ): ColumnDef.Single.WithTableMeta[ImagingModeRowWithResult, V, TableMeta] =
+    ColDef(id, accessor, columnNames.getOrElse(id, id.value))
+
+  private def columns(units: WavelengthUnits) =
+    given Display[BoundedInterval[Wavelength]] = wavelengthIntervalDisplay(units)
+    given Display[WavelengthDelta]             = wavelengthDeltaDisplay(units)
+    given Display[Wavelength]                  = wavelengthDisplay(units)
+
+    given Order[Angle] = Angle.AngleOrder
+
+    List(
+      column(InstrumentColumnId, row => ImagingModeRow.instrument.get(row.entry).longName)
+        .withCell(_.value: String)
+        .withColumnSize(Resizable(120.toPx, min = 50.toPx, max = 150.toPx))
+        .sortable,
+      column(TimeColumnId, _.totalItcTime)
+        .withHeader(progressingCellHeader("Time"))
+        .withCell: cell =>
+          itcCell(cell.row.original.result, TimeOrSNColumn.Time)
+        .withColumnSize(FixedSize(85.toPx))
+        .withSortUndefined(UndefinedPriority.Last)
+        .sortable,
+      column(SNColumnId, _.totalSN)
+        .withHeader(progressingCellHeader("S/N"))
+        .withCell: cell =>
+          itcCell(cell.row.original.result, TimeOrSNColumn.SN)
+        .withColumnSize(FixedSize(85.toPx))
+        .withSortUndefined(UndefinedPriority.Last)
+        .sortable,
+      column(FilterColumnId, row => ImagingModeRow.instrumentConfig.get(row.entry))
+        .withCell(_.value.filterStr)
+        .withColumnSize(FixedSize(69.toPx))
+        .sortableBy(_.filterStr),
+      column(LambdaColumnId, row => ImagingModeRow.filter.get(row.entry).wavelength)
+        .withHeader(s"λ ${units.symbol}")
+        .withCell(_.value.fold("-")(_.shortName))
+        .withColumnSize(FixedSize(75.toPx))
+        .sortable,
+      column(
+        DeltaLambdaColumnId,
+        row => ImagingModeRow.filter.get(row.entry).wavelengthRangeAndDelta
+      )
+        .withHeader(s"Δλ ${units.symbol}")
+        .withCell: cell =>
+          val (range, delta) = cell.value
+          <.span(delta.fold("-")(_.shortName))
+            .withOptionalTooltip(range.map(r => s"${r.shortName} ${units.symbol}"))
+        .withColumnSize(FixedSize(100.toPx))
+        .sortableBy(_._2),
+      column(FovColumnId, _.entry.fov)
```

Comment:
will we display if the filter is narrow/broad?

Reply:
We can. It wasn't in the mockup. I suppose it probably should be. It can be `Narrow` `Broad` or `Combination` for the imaging filters. (so far)